main.cpp

```diff
@@ -125,62 +161,41 @@ struct CarWash
  */
 
 
-/*
-1）Leetcode
-5 properties:
-    1)number of comments
-    2)number of submissions
-    3)number of solved questions
-    4)acceptance rate
-    5)overall score
-3 things it can do
-    1)write code
-    2)help others with their code
-    3)pin a thread
- */
-struct Leetcode
+
+struct Leetcode//1
 {
-    //1)number of comments
     int numberOfComments = 24;
-    //2)number of submissions
     int numberOfSubmissions = 4;
-    //3)number of solved questions
     int numberOfSolvedQuestions = 13;
-    //4)acceptance rate
     float acceptanceRate = 0.66f;
-    //5)overall score
     float overallScore = 87.5f;
-    //1)write code
-    void writeCode(int languageType);
-    //2)help others with their code
+    int writeCode(int languageType);
     void helpOthersWithTheirCode(int numberOfQuestion);
-    //3)pin a thread
     void pinAThread(int numberOfThread);
 };
-/*
-2)Safeway
-5 properties:
-    1)open time
-    2)discount
-    3)Customer's points
-    4)close time
-    5)number of parking spots
-3 things it can do
-    1)order grocery online
-    2)refund
-    3)collect coupons
- */
-struct Safeway
+
+int Leetcode::writeCode(int languageType)//function
+{
+    std::cout << "language type chosen: " << languageType << std::endl;
+    return 1;
+}
+
+void Leetcode::helpOthersWithTheirCode(int numberOfQuestion)
+{
+    std::cout << "Number of Questions " << numberOfQuestion << std::endl;
+}
+
+void Leetcode::pinAThread(int numberOfThread)
+{
+std::cout << "Number of Thread " << numberOfThread << std::endl;
+}
```

Comment:
Fix this indentation
Code inside of curly braces `{ }` should be indented 4 spaces
- [ ] revise